### PR TITLE
[MIG] account_move_line_tax_editable: Migration to 14.0 

### PR DESCRIPTION
**Pull request by Zar21**
Standard migration to 14.0

### CODE REVIEW

**Review by hildickethan — APPROVED** [review]
Functional review

**Review by HaraldPanten — APPROVED** [review]
LGTM

File: account_move_line_tax_editable/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Move Line Tax Editable",
+    "summary": "Allows to edit taxes on non-posted account move lines",
+    "version": "14.0.1.1.0",
```
Comment (pedrobaeza) [inline account_move_line_tax_editable/__manifest__.py:7]:
```suggestion
    "version": "14.0.1.0.0",
```

**Pull request merged**